README.md

```diff
@@ -7,8 +7,11 @@
 
 The Stellar Anchor SDK is a collection of projects that make easy to build financial applications on Stellar.
 
-* [core](./core) is the base package for implementing standardized anchor applications.
-* [payment-circle](./payment-circle) is a package that wraps a subset of the [Circle API](https://developers.circle.com/reference).
+* [core](./core): the base package for implementing standardized anchor applications.
+* [config-spring-property](./config-spring-property): Implementation of configuration management with [Spring ConfigurationProperties](https://docs.spring.io/spring-boot/docs/current/reference/html/features.html#features.external-config).
+* [data-spring-jdbc](./data-spring-jdbc): Implementation of data access with [Spring Data JDBC](https://spring.io/projects/spring-data-jdbc). 
```

Comment:
🤔  Is this package meant to use SQLite only? If so we should add `sqlite` to the package name.